src/main/java/AddTask.java

```diff
@@ -0,0 +1,72 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class AddTask {
+    public static void checkDescription(String[] userInputWords) throws
+            DukeExceptions.NoDescriptionException {
+        if (userInputWords.length == 1) {
+            DukeExceptions.throwNoDescriptionException();
+        }
+    }
+
+    public static Task addSpecialTask(String description) {
+        String[] userInputWords = description.split(" ");
+        try {
+            checkDescription(userInputWords);
+        } catch(DukeExceptions.NoDescriptionException e) {
+            String taskType = userInputWords[0];
+            PrintText.printWithLinebreak(taskType + " must have a description.");
+            return new Task("Not special task");
+        }
+        char type = Character.toUpperCase(userInputWords[0].charAt(0));
+        return new Task(description, type);
+    }
+
+    public static void taskListManager(ArrayList<Task> tasks) {
+        int index = DukeFile.latestIndex();
+        boolean isRunning = true;
+        while (isRunning) {
+            Scanner userInput = new Scanner(System.in);
+            String text = userInput.nextLine();
+            String[] userInputWords = text.split(" ");
+            switch (userInputWords[0]) {
+            case "bye":
+                isRunning = false;
+                break;
+            case "list":
+                PrintText.print(PrintText.LINEBREAK);
+                PrintTask.list(tasks);
+                PrintText.print(PrintText.LINEBREAK + "\n");
+                break;
+            case "mark":
+            case "unmark":
+                MarkTask.mark(userInputWords, tasks);
+                break;
+            case "todo":
+            case "deadline":
```

Comment:
add a //Fallthrough comment if there is no break statement within a case block, see the coding standards if you are unsure